src/modules/balance/handlers/handlerBalance.mixin.js

```diff
@@ -0,0 +1,75 @@
+import { getBalanceHistory } from '@/apollo/queries/wallets/wallets.graphql';
+import utils from 'web3-utils';
+import BigNumber from 'bignumber.js';
+import { Toast, ERROR } from '@/modules/toast/handler/handlerToast';
+
+export default {
+  name: 'HandlerBalance',
```

Comment:
rename this please to the HandlerBalance history and the file name it's self (this will help, avoid confusion)

Reply:
done!